README.md

```diff
@@ -412,6 +412,7 @@ Table of Contents
    * [elephantsql.com](http://www.elephantsql.com/) — PostgreSQL as a service, 20 MB free
    * [graphenedb.com](http://www.graphenedb.com/) — Neo4j as a service, up to 1,000 nodes and 10,000 relations free
    * [mlab.com](https://mlab.com/) — MongoDB as a service, 500 MB free
+   * [MongoDB Atlas](https://www.mongodb.com/cloud/atlas) — free tier gives 512 MB
```

Comment:
This is another:  they did mention that they offer a free tier like mLab

https://www.mongodb.com/cloud/atlas/compare